src/net/objects.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 The reone project contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <cstdint>
+
+namespace reone {
+
+namespace net {
+
+struct BaseStatus { };
+
+// dummy structure for testing
+struct CreatureStatus : public BaseStatus {
```

Comment:
libnet was intended to be lower level than that. Notion of creature state (especially equipment, faction and etc.) is game logic and belongs to libmp.

Reply:
I'll move `snapshot.h` and `snapshot.cpp` to libmp as well.